### PR TITLE
second registration fixed bugs

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/common/stores/applications/SsvWeb/Cluster.store.ts [hunk 1]
```diff
@@ -144,8 +145,15 @@ class ClusterStore extends BaseStore {
       burnRate: burnRate,
       balance: newBalance,
     });
+
+    const keys = Object.keys(cluster);
+    let camelKeysCluster: any = {};
+    keys.forEach((key: string) => {
+      camelKeysCluster[_.camelCase(key)] = cluster[key];
```
Comment (alexstolr) [inline src/app/common/stores/applications/SsvWeb/Cluster.store.ts:152]: You probably need to delete the old non camel case keys

**Review by alexstolr — APPROVED** [review]

File: src/app/common/stores/applications/SsvWeb/Cluster.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/SsvWeb/Cluster.store.ts:152]: no, I dont use in future cluster variable so don't need to do something with it

File: src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx [hunk 2]
```diff
@@ -108,10 +108,11 @@ const FundingNewValidator = () => {
     if (Number(depositSSV) > 0) return <Typography className={classes.PositiveDays}>(+{depositSSV} SSV)</Typography>;
   };
 
-  const moveToNextPage = () => {
+  const moveToNextPage = async () => {
```
Comment (IlyaVi) [inline src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx:111]: there is no await, are you sure this function needs to be async

**Review by IlyaVi — APPROVED** [review]

File: src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx:111]: +

**Pull request merged**